applications/CompressiblePotentialFlowApplication/python_scripts/define_embedded_wake_process_2d.py

```diff
@@ -0,0 +1,59 @@
+import KratosMultiphysics
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
+import KratosMultiphysics.MeshingApplication as MeshingApplication
+from KratosMultiphysics.CompressiblePotentialFlowApplication.define_wake_process_2d import DefineWakeProcess2D
+import time
+import math
+
+
+def Factory(settings, Model):
+    if( not isinstance(settings,KratosMultiphysics.Parameters) ):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+
+    return DefineEmbeddedWakeProcess(Model, settings["Parameters"])
+
+
+class DefineEmbeddedWakeProcess(KratosMultiphysics.Process):
+    def __init__(self, Model, settings):
+        # Call the base Kratos process constructor
+        KratosMultiphysics.Process.__init__(self)
+
+        # Check default settings
+        default_settings = KratosMultiphysics.Parameters(r'''{
+            "model_part_name": "",
```

Comment:
I would say that model_part_name has to always be there by default in all processes, but Im not sure @rubenzorrilla can you confirm?

Reply:
I am just thinking that from one point of view, it would be much more visual to whoever is reading the process to know which model_part needs to come as an input (although this we can describe also as a comment on top). From another point of view, what happens in the processes where more than one model part need to come in? (I know here is not the case anyway, just saw it an wanted to raise the question)